multidim-interop/impl/go/v0.27/Makefile

```diff
@@ -12,7 +12,7 @@ go-libp2p-${version}: go-libp2p-${version}.zip
 	unzip -o go-libp2p-${version}.zip
 
 go-libp2p-${version}.zip:
-	wget -O $@ "https://github.com/libp2p/go-libp2p/archive/v${version}.zip"
+	wget --secure-protocol TLSv1.3 -O $@ "https://github.com/libp2p/go-libp2p/archive/v${version}.zip"
```

Comment:
```suggestion
wget --secure-protocol TLSv1.3 -O $@ "https://github.com/libp2p/go-libp2p/archive/v${version}.zip"
```

Reply:
What is the diff here?

Reply:
Sorry, I've updated it

Reply:
why do you need the `=` ?

Reply:
That's how the [wget manual](https://www.gnu.org/software/wget/manual/html_node/HTTPS-_0028SSL_002fTLS_0029-Options.html) specifies it.